packages/manager/apps/hycu/src/tracking.constant.ts

```diff
@@ -0,0 +1,20 @@
+export const LEVEL2 = {
+  EU: {
+    config: {
+      level2: '98',
+    },
+  },
+  CA: {
+    config: {
+      level2: '98',
+    },
+  },
+  US: {
+    config: {
+      level2: '98',
+    },
+  },
+};
```

Comment:
```suggestion
export const LEVEL_2 = {
  EU: {
    config: {
      level2: '98',
    },
  },
  CA: {
    config: {
      level2: '98',
    },
  },
  US: {
    config: {
      level2: '98',
    },
  },
} as const;
```

Reply:
I kept `LEVEL2` to be consistent with other apps